docs/wiki/[User-Guide]-Deploy-Management-Resources.md

```diff
@@ -1,4 +1,29 @@
-## This page has been moved
```

Comment:
This "User Guide" page is here to catch old links and was legitimately deprecated.

The new user guide should be at `docs/wiki/[User-Guide]-Management-Resources.md` and the example page is at `docs/wiki/[Examples]-Deploy-Management-Resources.md`